src/styles/main.scss

```
@@ -1,3 +1,223 @@
-body {
+* {
   margin: 0;
 }
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+:root {
+  --blue: #00acdc;
+  --black: #000;
+  --grey: #616070;
+  --darkblue: #060b35;
+  --white: #fff;
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0 41px;
+  padding-top: 51px;
```

set width of container. it should be 
```suggestion
  padding: 50px 40px;
```
for setting foor cards in row container of cards should have width
it will be width of cards + distances between cards + horizontal paddings.
also do not set right margin for last card in row.